src/index.js

```diff
@@ -22,8 +22,16 @@ function setupLocalDataTrack() {
   let mouseDown;
   let mouseCoordinates;
 
-  window.addEventListener('mousedown', () => {
+  window.addEventListener('mousedown', event => {
     mouseDown = true;
+    const { pageX: x, pageY: y } = event;
+    mouseCoordinates = { x: x / window.innerWidth, y: y / window.innerHeight };
+
+    // Send the initial down coordinate
+    dataTrack.send(JSON.stringify({
+      mouseDown,
+      mouseCoordinates
+    }));
```

Comment:
I would prefer if we only sent coordinates from one of the event handlers, e.g., "mousemove". Also, what is the significance of sending the "initial down coordinate"? In general, it may never arrive. Better to stream full state, I think (as we were doing in "mousemove"), because then it doesn't matter if you miss a message.

Reply:
On the mobile clients we need a "begin" event so we know where to start our path. This is why I send the initial down event before sending the mouse move event.